tests/cassettes/InferenceClientVCRTest.test_zero_image_classification.yaml

```diff
@@ -0,0 +1,43 @@
+interactions:
```

Comment:
This file must be generated automatically (here the format is not correct). To do so, you must have `pytest-vcr` installed + run `pytest tests/test_inference_client.py -k "zero" --vcr-record=all` to record the HTTP call. Don't worry if it doesn't work for you, I can take of it :) Just be careful of not committing your HF token in the yaml file.

Reply:
This was extremely helpful! I could not find the command to generate this properly. With the typing changes, it passes all tests!